tests/protocol/mockwiredata.py

```diff
@@ -36,7 +37,13 @@
         "trans_cert": "wire/trans_cert",
         "test_ext": "ext/sample_ext-1.3.0.zip",
         "remote_access": None,
-        "in_vm_artifacts_profile": None
+        "in_vm_artifacts_profile": None,
+        "vm_artifacts_profile": "wire/in_vm_artifacts_profile_blob.json",
+        "vm_artifacts_profile_install": "wire/in_vm_artifacts_profile_blob_install.json",
+        "vm_artifacts_profile_no_extensions": "wire/in_vm_artifacts_profile_no_extensions.json",
+        "vm_artifacts_profile_no_ext_config": "wire/in_vm_artifacts_profile_no_ext_config.json",
```

Comment:
NIT: You dont need to add these test files here in the main array (which is shared with every test case), you can load them on a per demand basis whenever you want to test a particular case. Something like you're already doing - 
```
DATA_FILE_FAST_TRACK = DATA_FILE.copy()
DATA_FILE_FAST_TRACK["in_vm_artifacts_profile"] = "wire/in_vm_artifacts_profile_blob.json"
```

Reply:
changed